src/StreetNameRegistry/Municipality/ValueObjects/StreetNameHomonymAddition.cs

```diff
@@ -28,8 +28,10 @@ public StreetNameHomonymAddition(string homonymAddition, Language language)
 
         protected override IEnumerable<object> Reflect()
         {
-            yield return HomonymAddition;
+            yield return HomonymAddition.ToLowerInvariant();
             yield return Language;
         }
+
+        public override string ToString() => $"{HomonymAddition} ({Language})";
```

Comment:
The ToString was already overwritten in StreetNameName. So I've took over its format here. This means the format has changed `"{homonymaddition}, {language}"` to `"{homonymaddition} ({language})"`. Is this ok?